src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -213,20 +213,22 @@
 		margin: 0;
```

Comment:
You have to remember also about "box-shadow"

Reply:
Added box-shadow.